repository-data/development/src/main/resources/hcm-content/content/documents/corporate-website/general.yaml

```diff
@@ -648,6 +847,289 @@
+        hippo:values: []
+        jcr:primaryType: hippogallerypicker:imagelink
+      /website:sections[1]:
+        /website:html:
+          hippostd:content: <p>add section</p>
+          jcr:primaryType: hippostd:html
+        jcr:primaryType: website:section
+        website:headinglevel: Main heading
+        website:numberedList: false
+        website:title: need to add section
+        website:type: ''
+      /website:sections[2]:
+        /website:html:
+          hippostd:content: <p>add another sectionextraWideColumn</p>
+          jcr:primaryType: hippostd:html
+        jcr:primaryType: website:section
+        website:headinglevel: Main heading
+        website:numberedList: false
+        website:title: need to add another section
+        website:type: ''
+      /website:seosummary:
+        hippostd:content: ''
+        jcr:primaryType: hippostd:html
+      /website:summary:
+        hippostd:content: ''
+        jcr:primaryType: hippostd:html
+      /website:userContact:
+        /website:userContactnotes:
+          hippostd:content: ''
+          jcr:primaryType: hippostd:html
+        jcr:primaryType: website:userContact
+        website:userContactPhone: ''
+        website:userContactemail: hgfhfghfhf
+        website:userContactname: ''
+        website:userContactprimarycontact: false
+      /website:userNeed:
+        jcr:primaryType: website:userNeed
+        website:asA: ''
+        website:iNeedto: ''
+        website:soThat: ''
+      /website:userNeedandcontact:
+        /website:briefDescription:
+          hippostd:content: ''
+          jcr:primaryType: hippostd:html
+        /website:teampicker:
+          hippo:docbase: cafebabe-cafe-babe-cafe-babecafebabe
+          jcr:primaryType: hippo:mirror
+        jcr:primaryType: website:userNeedandcontact
+        website:portfolioCode: ''
+        website:responsibleTeam: ''
+        website:userNeedismet:
+        - ''
+      common:searchRank: 3
+      common:searchable: true
+      hippo:availability:
+      - live
+      hippostd:state: published
+      hippostdpubwf:createdBy: admin
+      hippostdpubwf:creationDate: 2020-10-05T13:40:16.994Z
+      hippostdpubwf:lastModificationDate: 2020-10-21T14:40:11.081Z
+      hippostdpubwf:lastModifiedBy: admin
+      hippostdpubwf:publicationDate: 2020-10-21T14:40:25.834Z
+      hippotranslation:id: 1f5aa496-06c6-43f7-9204-373bf896a881
+      hippotranslation:locale: en
+      jcr:mixinTypes:
+      - hippotaxonomy:classifiable
+      - mix:referenceable
+      jcr:primaryType: website:general
+      website:gossid: 0
+      website:metadata: []
+      website:navigationcontroller: withoutNav
+      website:shortsummary: ''
+      website:title: lesley's test general doc type
+      website:type: ''
+      website:wideMode: false
+    hippo:versionHistory: 357cb607-7a1b-4b86-b09f-c57a4e70ed0f
+    jcr:mixinTypes:
+    - hippo:versionInfo
+    - mix:referenceable
+    jcr:primaryType: hippo:handle
```

Comment:
Did you mean to commit the `lesleytestonly` page? If not you can probably reset this file.

Reply:
i am not quite sure how it works Geoff, I was told during induction, that development content do not get published to production, perhaps I have misunderstood? I would like to know how to keep things clean and simple...